src/sentry/utils/json.py

```diff
@@ -99,6 +99,10 @@ def dumps(value, escape=False, **kwargs):
     return _default_encoder.encode(value)
 
 
+def load(fp, **kwargs):
```

Comment:
This is unrelated, but added for API completeness of simplejson. I'm not sure how we didn't have this, but noticed it when I tried to do `json.load(fp)` and we didn't have it.